### PR TITLE
mgr/cephadm: provide upgrade completed/stopped message in orch upgrade status

### PR DESCRIPTION
**Pull request by adk3798**
The message won't be persistent across mgr fail overs but that shouldn't be a big deal. The main use case is just to have something that explicitly tells users the upgrade has completed for users repeatedly checking 'orch upgrade status' to see how the upgrade is going.

Also, swapped to using user given name for the upgrade target image in upgrade status messages as I think it's more readable.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/cephadm/upgrade.py [hunk 1]
```diff
@@ -91,6 +92,7 @@ def __init__(self, mgr: "CephadmOrchestrator"):
             self.upgrade_state: Optional[UpgradeState] = UpgradeState.from_json(json.loads(t))
         else:
             self.upgrade_state = None
+        self.upgrade_completion_message = ''  # for mypy
```
Comment (sebastian-philipp) [inline src/pybind/mgr/cephadm/upgrade.py:95]: not just for mypy. it is good practice to declare all members in `__init__`

```suggestion
        self.upgrade_completion_message = ''
```

**Review by sebastian-philipp — COMMENTED** [review]
is it really worth introducing the complexity for this message? This needs to be tested and maintained.

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2021-05-28_02:36:20-rados-wip-kefu-testing-2021-05-28-0853-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-05-28_13:33:45-rados-wip-kefu-testing-2021-05-28-1806-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-05-29_02:33:40-rados-wip-kefu-testing-2021-05-28-1806-distro-basic-smithi/

failures tracked by

- https://tracker.ceph.com/issues/45423
- https://tracker.ceph.com/issues/50151

**Comment by tchaikov** [comment]
@sebastian-philipp ping?

**Pull request closed without merging**